tests/functional/spec/components/checkbox-mutually-exclusive.spec.js

```diff
@@ -108,4 +116,29 @@ describe('Component: Mutually Exclusive Checkbox', function() {
     });
   });
 
+  describe('Given the user does selects any options on the optional page', function() {
+    it('They should see the interstitial page', function() {
+      return browser
+        .click(MandatoryCheckboxPage.noExtra())
+        .isSelected(MandatoryCheckboxPage.noExtra()).should.eventually.be.true
+        .click(MandatoryCheckboxPage.submit())
+        .click(OptionalCheckboxPage.cheddar())
+        .click(OptionalCheckboxPage.submit())
+        .getUrl().should.eventually.contain(CheeseInterstitialPage.pageName)
+        .click(CheeseInterstitialPage.submit())
+        .getUrl().should.eventually.contain(SummaryPage.pageName)
+    });
+  });
+
+  describe('Given the user does not select any options on the optional page', function() {
+    it('They should not see the interstitial page', function() {
+      return browser
+        .click(MandatoryCheckboxPage.noExtra())
+        .isSelected(MandatoryCheckboxPage.noExtra()).should.eventually.be.true
+        .click(MandatoryCheckboxPage.submit())
+        .click(OptionalCheckboxPage.submit())
+        .getUrl().should.eventually.contain(SummaryPage.pageName)
+    });
+  });
+
```

Comment:
Is the behaviour we're testing specific to a mutually exclusive checkbox? If not, I don't think it should be added to the mutually exclusive schema and associated tests.

Reply:
Fair point, I'll move it to a seperate schema.

Reply:
I've moved it to a test_routing_checkbox_set_not_set.json schema